tests/kat_sig_stfl.c

```diff
@@ -182,7 +182,7 @@ OQS_STATUS sig_stfl_kat(const char *method_name, const char *katfile) {
 		goto err;
 	}
 
-	if (!ReadHex(fp_rsp, secret_key, sig->length_secret_key, "sk = ")) {
+	if (!ReadHex(fp_rsp, secret_key->secret_key_data, sig->length_secret_key, "sk = ")) {
```

Comment:
OK for now but, we eventually will want to the serialize_key/deserialize_key to get and set the private key data.
secret_key->secret_key_data,
if (secret_key->serialize_key) {
        sk_buff =secret_key->serialize_key(secret_key);
        ....
}